# Sensor and command status
bool enable
bool hasGPS
bool hasIMU

# Waypoints
int16 waypoints
float64 tgtLon
float64 tgtLat

# headings
float64 desiredHeading
float64 deltaHeading

# distance
float64 linearError
